umh-core/pkg/fsm/base_manager.go

```
@@ -0,0 +1,543 @@
+					instanceName, TicksBeforeNextRemove-(m.managerTick-m.lastRemoveTick))
+				continue // Skip this removal for now, will be removed on a future tick
+			}
+
+			// Temporary logging
+			if instanceName == "golden-service" {
+				m.logger.Errorf("m.instances: %v, desiredState: %v", m.instances, desiredState)
+			}
+
+			// Otherwise, we need to remove the instance
+			m.logger.Debugf("instance %s is in state %s, starting the removing process", instanceName, instance.GetCurrentFSMState())
+			instance.Remove(ctx)
```

golangci-lint (1.64.8)

Error return value of `instance.Remove` is not checked

(errcheck)